azure-pipelines-dev.yml

```
@@ -0,0 +1,28 @@
+variables:
+- template: /eng/templates/azure-pipelines-shared-variables.yml
```

Nit: we tend to explicitly pin the template reference to the same commit / repo the pipeline is being run from by adding `@self` at the end of the filepath:
```suggestion
- template: /eng/templates/azure-pipelines-shared-variables.yml@self
```